dist/workflow-list.tpl.html.js

```diff
@@ -24,7 +24,11 @@ ngModule.run(['$templateCache', function ($templateCache) {
     '  <md-list-item ng-repeat="workflow in ctrl.workflows" ng-click="ctrl.selectWorkflow($event, workflow)" ng-class="{active: selected.id === workflow.id}" >\n' +
     '    <div class="md-list-item-text">\n' +
     '      <p>\n' +
-    '        {{workflow.title}}\n' +
+    '        {{workflow.title}} <br/>\n' +
+    '        <md-icon ng-show="workflow._syncStatus.type === \'failed\'" md-font-set="material-icons" class="material-icons sync-error-icon">\n' +
```

Comment:
`ng-if` drirective with workflow._syncStatusFaild param check? Think better create some scope attribute and play with it rather compare values directly in template